middleware/csrf/storage/memory.go

```diff
@@ -0,0 +1,77 @@
+package storage
```

Comment:
I think it is unnecessary to separate memory storage into another folder/package